Makefile

```diff
@@ -49,5 +49,10 @@ check_sample_files: ## Make sure each template in policies/templates has one sam
 check_format: ## Check that files have been formatted using opa fmt
 	@test $$(opa fmt -l lib/ validator/ | wc -l) -eq '0'
 
+.PHONY: audit
+audit:
+	@echo "Running config-validator audit ..."
+	@sh scripts/cft.sh -p ${PROJECT_ID} -o ${ORG_ID} -b ${CAI_BUCKET_NAME} -e ${EXPORT}
```

Comment:
@morgante I've noticed that I can't use both flags `-p` and `-o` (if one is empty, then cft scorecard will hang), so not sure how to structure this call here

Reply:
I'd say your wrapper script should only pass through which value is non-empty.

Reply:
Actually, scorecard doesn't support the `--organization` flag yet